packages/switchdin-superset-plugin-tariff-savings/src/transformProps.js

```diff
@@ -28,5 +25,5 @@ export default function transformProps(chartProps) {
     yAxisFormat,
     imageFile: imageSelectionControl,
     data: result,
-  }
+  };
```

Comment:
Having trouble following the rationale behind some of these changes. Is this the actual change?

Reply:
No they have some kind of git linter that did it auto-magically. The only change I have made is renaming "payload" to "queryData".

Reply:
I should have untangled the two changes.

Reply:
OK. No worries. That makes sense.